src/main/frontend/handler/editor.cljs

```diff
@@ -1951,10 +1951,12 @@
 
                    :else
                    true)]
+    (when editing-block
```

Comment:
Is this supposed to be `target-block`? If not, could you add a comment as to why `editing-block` must exist in order to save the target? It's a bit confusing since the coupling between the two isn't obvious

Reply:
This is just moving the logic from slightly further down; I had thought it meant "when there is currently a block being edited that isn't saved yet" but I think actually editing-block is always going to be truthy, so this perhaps does nothing.

The behavior I wanted was to, if there was a block being typed into with unsaved edits, then save those edits as a transaction before the paste transaction starts. I'll see if I can figure out how to accomplish that, since I realize now that doing it this way results in a blank undo entry before the paste if you paste while not actively editing.

Reply:
OK, I pushed a fix to actually do the logic I thought it was doing.